massgov/mayflower-patternlab
# Link list data cleanup

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)

## Description

This PR doesn't change anything render on the front end. It removes unused data for decorative links in link list pattern, and sets the title for see all link. It also renders sidebarHeading as an option of compHeading (backport mayflower-react)
 

## Related Issue / Ticket

- [JIRA issue]()
- [Github issue]()

## Steps to Test


1.  Stand up localhost and go to http://localhost:3000/?p=organisms-link-list
2. Inspect `see all` link and verify the title is being set
3. Pass `true` to sidebar in link-list.json and see the sidebarHeading rendering
